docs/development/building-packages-from-source.md

```diff
@@ -1,14 +1,14 @@
-(building-and-testing-packages-out-of-tree)=
+(building-packages-from-source)=
 
-# Building and testing Python packages out of tree
+# Building Python Packages from Source
 
 This is some information about how to build and test Python packages against
-Pyodide out of tree (for instance in your package's CI or for use with private
-packages).
+Pyodide, for instance in your package's CI or for use with private
+packages.
 
-Pyodide currently only supports Linux for out of tree builds, though there is a
-good change it will work in MacOS too. If you are using Windows, try Windows
-Subsystem for Linux.
+Pyodide currently only supports building packages in Linux environments officially,
+though there is a good change it will work in MacOS too.
```

Comment:
```suggestion
though there is a good chance it will work in MacOS too.
```

Note: Pyodide builds and runs fine for me on a Mac M1 (all `make` targets + `build-recipes`). The only issues I've faced are:

- `sed` (Mac `sed` doesn't like the bare `-i` flag, but using `gnu-sed` works fine)
- `pkg-config` resolves SSL to system libs rather than the ones in `pyodide/packages/ssl` - this can be worked around by tweaking the `hashlib` and `ssl` `meta.yaml` files to hardcode the paths. (Relative paths may just work fine here, I've not properly tried.)

Reply:
Thanks. Yes, we do build some packages in MacOS in our CI. But not all packages are built and tested, so we still have that note.